### ISSUE TITLE
Request - turn off NumLock

### ISSUE THREAD

**Issue by steve6375** [chenall/grub4dos#317]
command or utility to turn on/off NumLock

Not sure if this is possible, but  it would be nice to be able to turn off or turn on NumLock so that PageUp/PageDown/Home/End keys work on a laptop.

**Comment by a1ive** [comment]
BIOS or UEFI?
It's possible on UEFI, but I'm not sure about BIOS.
https://github.com/tianocore/edk2/blob/0ecdcb6142037dd1cdd08660a2349960bcf0270a/MdePkg/Include/Protocol/SimpleTextInEx.h#L116

**Comment by steve6375** [comment]
yes, UEFI will be great. I didn't think it was possible via legacy BIOS anyway without direct port I/O.
